t/01-pdk/08-response/14-get_raw_body.t

```diff
@@ -0,0 +1,37 @@
+use strict;
+use warnings FATAL => 'all';
+use Test::Nginx::Socket::Lua;
+do "./t/Util.pm";
+
+$ENV{TEST_NGINX_NXSOCK} ||= html_dir();
+
+plan tests => repeat_each() * (blocks() * 3);
+
+run_tests();
+
+__DATA__
+
+=== TEST 1: response.get_raw_body() gets raw body
+--- http_config eval: $t::Util::HttpConfig
+--- config
+    location = /t {
+        content_by_lua_block {
+            ngx.say("Hello, Content by Lua Block")
```

Comment:
Perhaps another test with a body large enough to span buffers? Otherwise much of the complexity won't be covered. Would `ngx.flush()` perhaps do it? In the jq plugin I did something like `string.rep("*", 1024 * 1024)`, which was enough to reliably cause buffering.

Reply:
yes, i was planning to add more tests, but wanted to open it for discussion already